### DISCUSSION TITLE
`AsDiscreted` and `Activationsd` support for batch of tensors?

### DISCUSSION THREAD

**Discussion by YerePhy** [Project-MONAI/MONAI#4349] [Q&A] [answered]
I have do some experiments ith theses clases and they not support batched images `NCHW[D]`. Well in `AsDiscreted`, `toech.argmax` is applied over `dim=0`. Is there something that supports batches of images implemented or batches are not supported?

**Comment by Nic-Ma** [comment, marked as answer]
Hi @YerePhy ,

We usually decollate a batch data and execute postprocessing transforms for every item, for more details please check:
https://github.com/Project-MONAI/tutorials/blob/main/modules/decollate_batch.ipynb

Thanks.

**Reply by YerePhy** [reply]
Thanks @Nic-Ma, you are faster than light with the answers hehe.
